### PR TITLE
add publishing rules for openshift/kubernetes repo

### PR DESCRIPTION
**Pull request by mfojtik**
@sttts @deads2k FYI.

I'm going to fetch the rules in openshift/kubernetes bot directly from master branch.

### CODE REVIEW

File: publishing-kube-rules.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/code-generator
+- destination: kubernetes-apimachinery
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/apimechinery
```
Comment (soltysh) [inline publishing-kube-rules.yaml]: typo: apim **a** chinery

File: publishing-kube-rules.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    source:
+      branch: master
+      dir: staging/src/k8s.io/metrics
+- destination: kubernetes-apiserver
+  branches:
+  - name: release-1.9.1
+    source:
+      branch: master
+      dir: staging/src/k8s.io/apiserver
+- destination: kubernetes-aggregator
```
Comment (soltysh) [inline publishing-kube-rules.yaml]: It's kubernetes-kube-aggregator: https://github.com/openshift/kubernetes-kube-aggregator

**Comment by mfojtik** [comment]
@soltysh typos fixed, thanks

**Comment by soltysh** [comment]
I'll leave it for David to do the final sign-off on this one.

**Comment by deads2k** [comment]
I think this is going to need fixing to pull from openshift/kubernetes and that logically we'll want to use the per-branch file, but I won't let perfection be the enemy of good.

/lgtm

**Comment by mfojtik** [comment]
Merging by hand as this is new file addition, does not need tests.

**Pull request merged**